File: app/src/main/java/com/xu/database/MapData.java
package com.xu.database;

public class MapData {
    private String key;
    private String value;

    public MapData(String key, String value){
        this.key = key;
        this.value = value;
    }

    public MapData(String key, int value){
        this.key = key;
        this.value = String.valueOf(value);
    }

    public String getKey() {
        return key;
    }

    public String getValue() {
        return value;
    }

    public String[] toArray(){
        return new String[] {key, value};
    }

    @Override
    public String toString(){
        return key + ">" + value;
    }
}

File: app/src/main/java/com/xu/database/MapData.kt
package com.xu.database

data class MapData (val key: String,var value: String) {
    constructor(key: String, value: Int) : this(key, value.toString())

    fun toArray(): Array<String?> {
        return arrayOf(key, value)
    }

    override fun toString(): String {
        return "$key>$value"
    }
}
